erpnext/accounts/doctype/process_statement_of_accounts/process_statement_of_accounts.py

```diff
@@ -240,8 +242,8 @@ def fetch_customers(customer_collection, collection_name, primary_mandatory):
 		if int(primary_mandatory):
 			if primary_email == "":
 				continue
-		elif (billing_email == "") and (primary_email == ""):
-			continue
+		# elif (billing_email == "") and (primary_email == ""):
```

Comment:
Why is this commented out?

Reply:
This line of code would continue the iteration of no email was found. Email is necessary only when I send it via email, not when I download the consolidated PDF. Hence, removed this code and added validation on email sending function

Reply:
We usually don't leave any commented out code, remove if its not needed

Reply:
Updated :)